op-node/rollup/driver/da.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 func SetDAClient(cfg celestia.CLIConfig) error {
-	client, err := celestia.NewDAClient(cfg.Rpc, cfg.AuthToken, cfg.Namespace)
+	client, err := celestia.NewDAClient(cfg.Rpc, cfg.AuthToken, cfg.Namespace, false)
```

Comment:
```suggestion
	client, err := celestia.NewDAClient(cfg.Rpc, cfg.AuthToken, cfg.Namespace, cfg.EthFallbackDisabled)
```

Reply:
We can use this config in `op-node` to disable interpreting fallback calldata, although it should never happen as the sequencer will never submit it.

Reply:
i see the comment. the rationale is that i've put false because it's not actually needed that flag in the derivation. i'm using (lazily) the DAClient as parameter store for the batcher rather than actually need that param in the client.

we need that derivation try to read from eth calldata when posted that way. imagine cases when you restart the node and you start back from the finalized state but in-between you enable the fallback for emergency purposes. so better to not force reading only from a source.

said that, might have sense to refactor da client in two parts like reader/writer in the future, since they might share some logic but as in this case they might not need all the parameters the other component need.

make sense @tuxcanfly ?

Reply:
OK, that makes sense. Yes, reads must be less restrictive than writes for all optional flags.